stdlib/contrib/rhajek/bigpanda/bigpanda.flux

```diff
@@ -46,12 +62,18 @@ sendAlert = (
     return http.post(headers: headers, url: url, data: json.encode(v: data))
 }
 
-// `endpoint` creates a factory function that creates a target function for pipeline `|>` to send alert to BigPanda for each table row.
-// `url` - string - base URL of [BigPanda API](https://docs.bigpanda.io/reference#alerts).
-// `token` - string - BigPanda authorization Bearer token
-// `appKey` - string - BigPanda App Key.
-// The returned factory function accepts a `mapFn` parameter.
-// The `mapFn` must return an object with all properties defined in the `sendAlert` function arguments (except url, apiKey and appKey).
+// endpoint sends alerts to [BigPanda](https://www.bigpanda.io/) using data from input rows.
+// 
+// ## Parameters
+// 
+// - url: BigPanda [alerts API URL](https://docs.bigpanda.io/reference#alerts-how-it-works).
+//   Default is the value of the `bigpanda.defaultURL` option.
+// - token: (Required) [BigPanda API Authorization token (API key)](https://docs.bigpanda.io/docs/api-key-management).
+// - appKey: (Required) BigPanda [App Key](https://docs.bigpanda.io/reference#integrating-monitoring-systems).
```

Comment:
Why are we adding the `(Required)` annotations here? That should already be part of the type signature and explicitly represented in the docs structures.